overlord/servicestate/quotas.go

```diff
@@ -29,6 +29,16 @@ import (
 
 var ErrQuotaNotFound = errors.New("quota not found")
 
+// AllQuotas returns all currently tracked quota groups in the state. They are
+// validated for consistency using ResolveCrossReferences before being returned.
+// API request should only succeed if the cgroups are enabled in the system
+func AllQuotasForDaemon(st *state.State) (map[string]*quota.Group, error) {
+	if memoryCGroupError != nil {
+		return nil, memoryCGroupError
```

Comment:
Here you could change this to `return nil, fmt.Errorf("cannot retrieve quota information: %v", memoryCGroupError)`